src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.Gss.cs

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+using size_t = System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class libgssapi
+    {
+        [DllImport(Interop.Libraries.SecurityNative)]
+        internal static extern Status GssReleaseBuffer(
```

Comment:
We've now moved to using the Library name as a prefix for all shim functions, e.g. SecurityNative_GssReleaseBuffer.